packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -127,7 +135,13 @@ const SettingsSidebarContainer = ({ className = "" }) => {
 
   return (
     <nav
-      className={`no-scrollbar flex w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll py-3 px-2 ${className}`}
+      className={classNames(
```

Comment:
Instead of having 2 instances of the nav (for mobile and desktop), we now use this boolean to add a few conditional styles to toggle the div between its desktop and mobile state 🔥